net/openvswitch/Makefile

```diff
@@ -33,7 +33,7 @@ PKG_INSTALL:=1
 PKG_MAINTAINER:=Yousong Zhou <yszhou4tech@gmail.com>
 
 include $(INCLUDE_DIR)/package.mk
-include ../../lang/python/python-host.mk
+include ../../lang/python/python3-host.mk
```

Comment:
python[3]-host.mk gets included by python[3]-package.mk
not a problem to leave it

Reply:
;) That's implementation details.  Let's include it explicitly